.gitignore

```diff
@@ -1,33 +1,35 @@
-.DS_Store
-logs
+*.*-e
 *.log
-pids
 *.pid
 *.seed
-lib-cov
-coverage
+*.sublime-project
+*.sublime-workspace
+.DS_Store
+.DS_Store
 .grunt
-build/Release
-node_modules
 .lock-wscript
-.DS_Store
+.map
+.project
+.settings
+Resources
+Thumbs.db
+Web-SDK
+Web-SDK/
```

Comment:
sure thing.

In different systems symlink are treated differently, for example, github treats this Web-SDK as a solid file, and thus Web-SDK (Web-SDK/ doesn't work); but on linux, mac included, ln links are treated as folders, and thus Web-SDK/ (Web-SDK doesn't work).